presto-product-tests/bin/product-tests-suite-3.sh

```diff
@@ -22,5 +22,11 @@ presto-product-tests-launcher/bin/run-launcher test run \
     -- -g storage_formats,cli,hdfs_impersonation,authorization -x iceberg,"${DISTRO_SKIP_GROUP}" -e "${DISTRO_SKIP_TEST}" \
     || suite_exit_code=1
 
+# Smoke run af a few tests in environment with dfs.data.transfer.protection=true. Arbitrary tests which access HDFS data were chosen.
+presto-product-tests-launcher/bin/run-launcher test run \
+    --environment singlenode-kerberos-hdfs-impersonation-with-data-protection \
```

Comment:
How does this environment is different to `singlenode-kerberos-hdfs-impersonation-with-wire-encryption`

Reply:
When it comes to hadoop config:

```
    <property>
        <name>dfs.data.transfer.protection</name>
        <value>privacy</value>
    </property>
```

instead of
```
    <property>
        <name>dfs.encrypt.data.transfer</name>
        <value>true</value>
    </property>
```

Reply:
I wish I could know that from reading the code...

Reply:
Thanks!